assets/src/dashboard/templates/raw/beauty.json

```diff
@@ -0,0 +1,5275 @@
+{
```

Comment:
Yea I'm open to whatever here, was just trying to get it to work while I was making it.

What are you thinking for passing the json into this script?

I agree that it's kind of just a dumping ground, wondering if it will be nice tho when someone wants to edit a template & they didn't create it on their machine, they can just import the `templates/raw` json into the editor and start working on it

Reply:
Another option is that `templates/raw` is the source of truth and we generate every template in CI on commit. might add considerable overhead tho for the build process which would be narsty.

This ticket might align with some of the work being done here too:
https://app.zenhub.com/workspaces/web-stories-5e94d3aced449034e1e9b226/issues/google/web-stories-wp/2120

Reply:
How would they import the raw json into the editor?  If it involves any javascript, then they should be able to import the results of the calling the template function in the same way, right?  And if they were to import the raw json directly (without fixing the image links), non of the images will load because they're all housed on someone else's machine.

I'm not against the idea of making the `json` files the source of truth, but it would be tricky with CI and stuff because we'd either have to commit the generated files and mark them with "DO NOT EDIT", or we don't commit them and just make sure they're always generated before the app loads (this includes when the app is released as a wordpress plugin)... 

In the end, the templates will _not_ be housed by the plugin at all, so I'd opt for the easier solution.

Reply:
Yup. I'm following now. thanks for the explanation

Reply:
JSON as the source of truth seems easier to handle to me: one can easily copy/paste it from the editor (using the debug tool we have) and we can better automated their migration (#2140).

`beauty.js` would then look like this:

```
import templateData from './beauty.json'; // returns an object thanks to babel-plugin-inline-json-import

export default function (imageBaseUrl) {
	// loop through templateData.pages[].elements[].resource and fix all URLs.

   return ${formattedTemplateData};
 }
```

Reply:
alright, new changes should just do all this calculation on runtime based off the outputted story json